dvc/command/experiments.py

```diff
@@ -1112,9 +1112,9 @@ def add_parser(subparsers, parent_parser):
         nargs="?",
         default=None,
         help=(
-            "Optional Git remote name or Git URL. If provided, experiments "
-            "from the specified Git repository will be listed instead of "
-            "local experiments."
+            "Optional Git remote name or a valid Git URL. If provided, "
+            "experiments from the specified Git repository will be "
+            "listed instead of local experiments."
```

Comment:
Small copy edit
```suggestion
            "Optional Git remote name or valid Git URL. If provided, "
            "experiments from the specified repository will be "
            "listed instead of local ones."
```